lib/document/document-converter.js

```diff
@@ -486,10 +486,22 @@ ${faDom.css()}
     if (node.getDocumentTitle()) {
       const doc = node.getDocument()
       if (this.hasTitlePage(doc)) {
-        return `<div id="cover" class="title-page front-matter">
-  <h1>${node.getDocumentTitle()}</h1>
-  <h2>${node.getDocument().getAuthor()}</h2>
+        const docTitle = node.getDocument().getDocumentTitle({ partition: true })
+        //If the document has a subtitle, we split title in h1 / subtitle in h2
+        if (docTitle.hasSubtitle()){
+          return `<div id="cover" class="title-page front-matter">
```

Comment:
To avoid duplicating the outer HTML `<div>`, you can do:

```js
const titlePageContent = /* ... */
// ...
return `<div id="cover" class="title-page front-matter">${titlePageContent}</div>`
```